src/components/pages/Story1/index.tsx

```diff
@@ -4,30 +4,48 @@ import Wireframe from '../../../assets/wireframe.png';
 import Layout, { SceneProps } from '../../shared/Layout';
 import test from '../../sketches/test';
 import test2 from '../../sketches/test2';
+import test3 from '../../sketches/test3';
+import test4 from '../../sketches/test4';
```

Comment:
i would like more descriptive names for the scenes.. the first couple I did should also have better names too